config/jobs-chromeos.yaml

```diff
@@ -376,13 +378,26 @@ _anchors:
     kcidb_test_suite: fluster.v4l2
 
   watchdog-reset: &watchdog-reset-job
-    template: watchdog-reset.jinja2
+    template: generic.jinja2
     kind: job
     params: &watchdog-reset-job-params
+      test_method: watchdog-reset
       bl_message: 'coreboot-'
       wdt_dev: 'watchdog0'
     kcidb_test_suite: kernelci_watchdog_reset
 
+  wifi-basic: &wifi-basic-job
+    template: generic.jinja2
+    kind: test
```

Comment:
Shouldn't that be `job`?

Reply:
@a-wai indeed, I missed this. Fixed now!